spring-integration-jms/src/main/java/org/springframework/integration/jms/config/JmsChannelFactoryBean.java

```diff
@@ -48,6 +48,8 @@
 import org.springframework.util.StringUtils;
 
 /**
+ * A {@link AbstractFactoryBean} implementation for creating {@link AbstractJmsChannel} instances.
```

Comment:
The English language requires an `An` article before words starting with vowels.